Project.toml

```
@@ -74,7 +74,7 @@ Requires = "1.1"
 SciMLBase = "1.90, 2"
 Setfield = "0.8, 1"
 SimpleUnPack = "1.1"
-StartUpDG = "0.17"
+StartUpDG = "0.17.6"
```

The default compat setting is as if you'd used the [caret specifier](https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers) `^`. That is, your compat settings says "[0.17.6, 0.18.0)".

You should use the [hyphen specifier](https://pkgdocs.julialang.org/v1/compatibility/#Hyphen-specifiers),
```suggestion
StartUpDG = "0.17 - 0.17.6"
```

to restrict to versions starting with the current compat version 0.17 and up to (and including) 0.17.6.